lib/doctor/Doctor.ts

```diff
@@ -245,14 +249,15 @@ export class Doctor {
       });
     }
 
-    if (latestCLIVersion !== pkg.version) {
+    if (latestCLIVersion !== pkg.version && nextCliVersion !== pkg.version) {
+      const onNextTag = pkg.version.includes('beta');
```

Comment:
This is the best way I know of to check if they are on the `next` tag, it doesn't always have latest, but that is usually when we publish to `latest` and the update the `next` tag to point to the same thing.